converter/src/test/resources/dd-980.xml.morph.result.json

```diff
@@ -0,0 +1,111 @@
+[ {
+  "http://data.slub-dresden.de/datamodels/DataModel-e5f5640f-767e-4019-a4f7-fb8c6ee51903/records/7605b78a-bc15-4aeb-8192-21fd70b7d01a" : [ {
+    "http://www.exlibrisgroup.com/xsd/primo/primo_nm_bib#sort" : [ {
+      "http://www.exlibrisgroup.com/xsd/primo/primo_nm_bib#title" : [ {
+        "http://www.w3.org/1999/02/22-rdf-syntax-ns#value" : "VBA mit Access"
+      }, { } ]
```

Comment:
ok, I think that this was, because the record id was always generated ... - so it should be fine ;)